src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -16,7 +16,7 @@
      * @param type - any class of Machine sub class. For example: Truck.class or Bulldozer.class
      * @return the list of machines
      */
-    List<Object> getAll(Class type);
+    List<T> getAll(Class type);
```

Comment:
It is better to parametrize `Class` so there will be only a class that we require with no mistakes